aptos-move/framework/aptos-framework/sources/account/account.move

```diff
@@ -327,7 +366,13 @@ module aptos_framework::account {
 
     #[view]
     public fun get_authentication_key(addr: address): vector<u8> acquires Account {
-        borrow_global<Account>(addr).authentication_key
+        if (resource_exists_at(addr)) {
+            borrow_global<Account>(addr).authentication_key
+        } else if (features::is_default_account_resource_enabled()) {
```

Comment:
Did we think it through thoroughly how this will interact with signature verification? Does this work for all key types? What about AA or DAA?

Reply:
default key type is just the ed25519. It's like the auto account creation by David.
For AA or DAA it's not relevant...